pkgs/top-level/all-packages.nix

```diff
@@ -10490,9 +10490,17 @@ with pkgs;
 
   lidarr = callPackage ../servers/lidarr { };
 
-  limesuite = callPackage ../applications/radio/limesuite {
-    inherit (darwin.apple_sdk.frameworks) GLUT;
-  };
+  inherit ({
```

Comment:
What is the purpose of the inherit statement?

Reply:
If I don't do that the CI complains about the new package, because it's not in pkgs/by-name. This is the recommend way of dealing with this:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#recommendation-for-new-packages-with-multiple-versions

Reply:
Thanks for the hint!